examples/fastapi/arcade_example_fastapi/main.py

```diff
@@ -12,14 +12,14 @@
 
 app = FastAPI()
 
-actor_secret = os.environ.get("ARCADE_ACTOR_SECRET")
+actor_secret = os.environ["ARCADE_ACTOR_SECRET"]
```

Comment:
Standardized on the syntax used a few lines above.

Accessing via key will force an error if the key does not exist (rather than fail silently), which is desired in this case because it provides immediate feedback.